neurobooth_os/iout/resplit_xdf.py

```diff
@@ -0,0 +1,353 @@
+"""
+This file splits an XDF file into constituent HDF5 files. It is meant to be called on the cluster without a full
+installation of Neurobooth-OS.
+"""
+
+import os
+import re
+import argparse
+import datetime
+import importlib
+from typing import NamedTuple, List, Dict, Optional, Any, Callable, ClassVar
+
+import yaml
+import psycopg2 as pg
+from pydantic import BaseModel
+
+import neurobooth_os.config as cfg
+import neurobooth_os.iout.split_xdf as xdf
+
+
+class SplitException(Exception):
+    """For generic errors that occur when splitting an XDF file."""
+    pass
+
+
+class HDF5CorrectionSpec(BaseModel):
+    marker: Optional[str] = None
+    devices: Dict[str, str] = {}
+
+    @staticmethod
+    def load(path: str) -> 'HDF5CorrectionSpec':

```

Comment:
i guess i forgot to ask this question, but why is the return type here quoted?  No need to do anything, just trying to understand

Reply:
never mind. found the answer in google once i found the right search terms

Reply:
I think in later versions of Python the quotes may not be necessary, but for 3.8 it's the syntactically correct way to do the type annotations when you need to refer to the class itself.